instaloader/structures.py

```diff
@@ -936,6 +936,11 @@ def followers(self) -> int:
     def followees(self) -> int:
         return self._metadata('edge_follow', 'count')
 
+    @property
+    def bio_links(self) -> List[str]:
```

Comment:
It seems the `bio_links` structure provides much more than just the URLs, e.g. a `title` property that might be interesting as well. I would prefer this method to also return the other information.

```json
[
  {
    "title": "",
    "url": "http://taylorswift.com",
    "link_type": "external"
  },
  {
    "title": "THE TORTURED POETS DEPARTMENT 🤍",
    "url": "https://taylor.lnk.to/thetorturedpoetsdepartment",
    "link_type": "external"
  },
  {
    "title": "Taylor Swift | The Eras Tour concert film",
    "url": "https://taylor.lnk.to/TSTheErasTourFilm",
    "link_type": "external"
  }
]
```